### PR TITLE
Issue #1584 - Add support for scanning config properties

### PR DESCRIPTION
**Pull request by arturdzm**

### CODE REVIEW

File: dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/AnnotationScanner.java [hunk 1]
```diff
@@ -166,19 +167,31 @@ private String findServletMappingForApp(String appClassName) throws UnableToAdap
...
+        Set<String> restAPIClasses = null;
+
+        try {
+            annotationTargets = webAnnotations.getAnnotationTargets();
+            restAPIClasses = ANNOTATION_CLASS_NAMES.stream().flatMap(anno -> annotationTargets.getAnnotatedClasses(anno,
+                                                                                                                   AnnotationTargets_Targets.POLICY_SEED).stream()).collect(Collectors.toSet());
+            Tr.event(tc, "Found annotated classes: ", restAPIClasses);
+
+        } catch (UnableToAdaptException e) {
+            e.printStackTrace();
```
Comment (leochr) [inline dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/AnnotationScanner.java]: Should this be changed to Tr.debug / Tr.event statement?

**Review by leochr — APPROVED** [review]
@arturdzm looks good. Just a minor clarification about e.printStackTrace() statement.

File: dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/AnnotationScanner.java (hunk 1 above)
Comment (arturdzm) [reply dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/AnnotationScanner.java]: Thanks, added Tr.event instead

Comment (leochr) [reply dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/AnnotationScanner.java]: looks good. thanks.

**Pull request merged**